src/Microsoft.VisualBasic/src/packages.config

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="System.Collections" version="4.0.10-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Diagnostics.Debug" version="4.0.10-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Dynamic.Runtime" version="4.0.10-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Globalization" version="4.0.10-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.IO" version="4.0.10-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Linq" version="4.0.0-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Linq.Expressions" version="4.0.10-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.ObjectModel" version="4.0.10-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Reflection" version="4.0.10-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Reflection.Extensions" version="4.0.0-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Reflection.Primitives" version="4.0.0-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Reflection.TypeExtensions" version="4.0.0-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Resources.ResourceManager" version="4.0.0-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Runtime" version="4.0.20-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Runtime.Extensions" version="4.0.10-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Runtime.Handles" version="4.0.0-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Runtime.InteropServices" version="4.0.20-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Text.Encoding" version="4.0.10-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Threading" version="4.0.10-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Threading.Tasks" version="4.0.10-beta-22405" targetFramework="portable-net45+win" />
```

Comment:
For some reason package manager in VS picked this version. I'll change this manually.